### PR TITLE
Exclude deleted boards in the selection for target

### PR DESCRIPTION
**Pull request by luka-nextcloud**
Signed-off-by: Luka Trovic <luka@nextcloud.com>

* Resolves: #2554
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] Sign-off message is added to all commits
- [ ] Tests (unit, integration, api and/or acceptance) are included
- [x] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: src/components/cards/CardMenu.vue [hunk 1]
```diff
@@ -129,6 +129,9 @@ export default {
 		isCurrentUserAssigned() {
 			return this.card.assignedUsers.find((item) => item.type === 0 && item.participant.uid === getCurrentUser()?.uid)
 		},
+		activeBoards() {
+			return this.$store.getters.boards.filter((item) => item.deletedAt === 0)
```
Comment (juliushaertl) [inline src/components/cards/CardMenu.vue:133]: For additional improvement, it could also filter out archived boards here:

```suggestion
			return this.$store.getters.boards.filter((item) => item.deletedAt === 0 && item.archived === false)
```

File: src/components/navigation/AppNavigationBoard.vue [hunk 2]
```diff
@@ -272,9 +272,7 @@ export default {
 							.then(() => {
 								this.loading = false
 								this.deleted = true
-								this.undoTimeoutHandle = setTimeout(() => {
-									this.$store.dispatch('removeBoard', this.board)
-								}, 7000)
+								this.$store.dispatch('removeBoard', this.board)
```
Comment (juliushaertl) [inline src/components/navigation/AppNavigationBoard.vue:275]: I guess this was for making testing easier, but please revert this change as the undo option should still be available.

Comment (luka-nextcloud) [reply src/components/navigation/AppNavigationBoard.vue:275]: @juliushaertl When we delete a board, we need to wait for 7s which is too long? Is there any reason for that?

Comment (juliushaertl) [reply src/components/navigation/AppNavigationBoard.vue:275]: UX-wise the idea is that delete is a destructive action and since we currently have nothing like a trashbin for deleted boards, the timeout will allow you to still undo this for some seconds. Lets not change that for now, but we could look into the deletion as an enhancement in the future.

**Review by juliushaertl — APPROVED** [review]
